composer.json

```diff
@@ -16,7 +16,9 @@
         "symfony/security-core": "^5.2|^6.2",
         "symfony/serializer": "^5.2|^6.2",
         "twig/twig": "^2.4|^3.0",
-        "symfony/translation": "^5.2|^6.2"
+        "symfony/translation": "^5.2|^6.2",
+        "jms/translation-bundle": "^1.8",
+        "doctrine/annotations": "^2.0"
```

Comment:
I don't remember seeing this or the dep above added in this PR. Was it to ensure these are there since they are implemented somewhere? If the implementations are optional, they should probably go into suggest, rather than requirements.

Reply:
This is in `src/Grid/Mapping/Driver/Annotation.php:33`

Reply:
And the translation stuff is in `src/Translation/ColumnTitleAnnotationTranslationExtractor.php:8`